redaxo/src/core/fragments/core/top.php

```diff
@@ -10,12 +10,14 @@
 
     foreach ($this->cssFiles as $media => $files) {
         foreach ($files as $file) {
-            echo "\n" . '    <link rel="stylesheet" type="text/css" media="' . $media . '" href="' . $file . '" />';
+            $path = rex_path::base(rex_path::absolute($file));
+            echo "\n" . '    <link rel="stylesheet" type="text/css" media="' . $media . '" href="' . $file . '?buster='. filemtime($path) .'" />';
```

Comment:
@phoebusryan könnte dieser neue parameter probleme machen mit z.b. minify da ggf. dort auch url parameter verwendet werden ? (rein ins blaue vermutet)

falls in $this->cssFiles urls enthalten sein könnten die bereits ein `?` enthalten müsste man diese hier noch behandeln.